src/Executables/Game/World/Entities/PlayerEntity.cs

```diff
@@ -269,6 +270,26 @@ private void CalculateDefence()
             // todo add defence bonus from quests
         }
 
+        private void CalculateMovement()
+        {
+            MovementSpeed = 150;
+            float modifier = 0;
+            foreach (var slot in Enum.GetValues<EquipmentSlots>())
+            {
+                var item = Inventory.EquipmentWindow.GetItem(slot);
+                if (item == null) continue;
+                var proto = _itemManager.GetItem(item.ItemId);
+                if (proto?.Type != (byte)EItemType.Armor) continue;
+                modifier += (byte)proto.Applies[0].Value;
```

Comment:
We should check apply value by its type, assuming index `0` would apply `MOVSPEED` is partially wrong. Could you check apply type with `EApplyType` enum please?

Reply:
Thanks, check has been added